Build failure with GHC 9.0.1

```
src/Text/TeXMath/Readers/TeX/Macros.hs:106:40: error:
    • Couldn't match type: forall st (m1 :: * -> *) s1.
                           Stream s1 m1 Char =>
                           ParsecT s1 st m1 T.Text
                     with: ParsecT s u m T.Text
      Expected: Macro -> ParsecT s u m T.Text
        Actual: Macro
                -> forall st (m :: * -> *) s.
                   Stream s m Char =>
                   ParsecT s st m T.Text
    • In the first argument of ‘map’, namely ‘macroParser’
      In the first argument of ‘choice’, namely ‘(map macroParser ms)’
      In the expression: choice (map macroParser ms)
    • Relevant bindings include
        tok :: ParsecT s u m T.Text
          (bound at src/Text/TeXMath/Readers/TeX/Macros.hs:104:11)
    |
106 |                   choice [ choice (map macroParser ms)
    |                                        ^^^^^^^^^^^
```

--- a/src/Text/TeXMath/Readers/TeX/Macros.hs
+++ b/src/Text/TeXMath/Readers/TeX/Macros.hs
@@ -103,7 +103,7 @@ applyMacrosOnce ms s =
        Left _  -> Nothing
     where tok = try $ do
                   skipComment
-                  choice [ choice (map macroParser ms)
+                  choice [ choice (map (\m -> macroParser m) ms)
                          , T.pack <$> ctrlseq
                          , T.pack <$> count 1 anyChar ]
 

